### PR TITLE
WINDUP-871 Migration Issues Report and Application List fixed hoverin…

### PR DESCRIPTION
**Pull request by mareknovotny**
…g description,

WINDUP-857 replaced hard-coded report titles with real report names from datamodel

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1294/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/667/

File: reporting/impl/src/main/resources/reports/templates/index.ftl [hunk 1]
```diff
@@ -79,9 +79,17 @@
             <div class="row">
                 <div class="page-header">
                     <h1>
-                        <div class="main">Application list</div>
+                        <div class="main"
+                        onmouseover="$(this).parent().parent().addClass('showDesc')"
+                        onmouseout=" $(this).parent().parent().removeClass('showDesc')"
```
Comment (mbriskar) [inline reporting/impl/src/main/resources/reports/templates/index.ftl]: is this the description questionmark functionality?

Comment (jsight) [inline reporting/impl/src/main/resources/reports/templates/index.ftl]: Yes

**Pull request merged**